docs/architecture/grpc-for-wcf-developers/kubernetes.md

```diff
@@ -33,39 +33,39 @@ metadata:
   # Object properties
 ```
 
-The `apiVersion` property is used to specify which version (and which API) the file is intended for. The `kind` property specifies the kind of object the YAML represents. The `metadata` property contains object properties such as `name`, `namespace`, or `labels`.
+The `apiVersion` property is used to specify which version (and which API) the file is intended for. The `kind` property specifies the kind of object the YAML represents. The `metadata` property contains object properties like `name`, `namespace`, and `labels`.
```

Comment:
This phrasing is a little confusing since `name` is mandatory property on `metadata`, but namespace is optional. Both AND and OR seem slightly misleading to me.

Reply:
To me, "and" makes more sense than "or" because we're just talking about what the property contains. Do you want to add clarity about which are optional and which are required?

Reply:
Let's not add complexity here. I was being extremely pedantic here, but on second reading I'm fine with `and`.